packages/hint/src/lib/cli/analyze.ts

```diff
@@ -22,8 +22,8 @@ import { installPackages } from '../utils/npm';
 import * as insights from '../utils/app-insights';
 import { FormatterOptions, IFormatter } from '../types/formatters';
 import loadHintPackage from '../utils/packages/load-hint-package';
+import boxen from 'boxen';
```

Comment:
Let's move this up to the group of imports coming from other packages (after `import * as async from 'async';`).

Reply:
Right! let me do that